test/e2e-secret-tests.sh

```diff
@@ -45,6 +45,14 @@ function export_variable() {
 # Setup resources common to all eventing tests.
 function test_setup() {
   pubsub_setup "secret" || return 1
+
+  # Authentication check test for BrokerCell. It is used in integration test in secret mode.
+  # We do not put it in the same place as other integration tests, because this test can not run in parallel with others,
+  # as this test requires the entire BrokerCell to be non-functional.
+  if [[ -v ENABLE_AUTH_CHECK_TEST && $ENABLE_AUTH_CHECK_TEST == 0 ]]; then
```

Comment:
Do I read it correctly that if ENABLE_AUTH_CHECK_TEST is 0, the test will run. I would expect 0 means disabled...

Reply:
Messed up with return 0 and return 1 in the script. To avoid that, I'll change it to be "true" in test-infra.

Reply:
For bash you can also simply check if `"$ENABLE_AUTH_CHECK_TEST" == "1"` as undefined var will be empty IIUC.

Reply:
I would like to keep `-v ENABLE_AUTH_CHECK_TEST `, in case there might return some unexpected error